source/services/reddit.js

```diff
@@ -1,7 +1,20 @@
 /**
  * Reddit service provider
+ *
+ * Notes:
+ * 1) How are shares counted?
+ * The number of up votes minus the number of down votes for a specific URL submitted on Reddit
+ * `children[0].data.score` - contain pre calculated value
+ *
+ * 2) Api request may failed with CORB (Cross-Origin Read Blocking) error for work need use jsonp callback
```

Comment:
I'm not sure I get this sentence, is it "An API request may fail because of CORB so we need to use JSONP to make it work"?

Reply:
Yes that's right. Update comment in 532a9eb